<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMainClass</class>
 <widget class="QMainWindow" name="ClientMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>799</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>IStrategizer BWAPI Client</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="mainLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QTabWidget" name="tbMainView">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <widget class="QWidget" name="tbEngineStatus">
         <attribute name="title">
          <string>Status</string>
         </attribute>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>11</y>
            <width>581</width>
            <height>571</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="engineStatusLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item row="9" column="0">
            <widget class="QLabel" name="lblWorkersCount">
             <property name="text">
              <string>Workers Count</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="lblGameCyclesData">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lblGameCycles">
             <property name="text">
              <string>Game Cycles</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="lblWorkersCountData">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="7">
            <widget class="QTableWidget" name="tblWorkerState">
             <property name="font">
              <font>
               <family>Tahoma</family>
               <pointsize>8</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <property name="gridStyle">
              <enum>Qt::NoPen</enum>
             </property>
             <property name="cornerButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="4">
            <widget class="QLabel" name="lblCurrUsedMemoryData">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="lblFPSData">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="lblStartUsedMemory">
             <property name="text">
              <string>Start Used Memory (KB)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLabel" name="lblStartMemoryData">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QLabel" name="lblCurrUsedMemory">
             <property name="text">
              <string>Current Used Memory (KB)</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="lblFPS">
             <property name="text">
              <string>FPS</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="7">
            <widget class="Line" name="line">
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="lineWidth">
              <number>7</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QLabel" name="lblDeltaMemory">
             <property name="text">
              <string>Delta Memory (KB)</string>
             </property>
            </widget>
           </item>
           <item row="6" column="4">
            <widget class="QLabel" name="lblDeltaMemoryData">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tbPlanner">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <attribute name="title">
          <string>Planner</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="plannerLayout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tbPlanHistory">
         <attribute name="title">
          <string>Plan History</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="planHistoryLayout">
            <item row="0" column="0">
             <widget class="QSlider" name="sldHistoryFrame">
              <property name="maximum">
               <number>1</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBothSides</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tbBuildingIM">
         <attribute name="title">
          <string>Building Data IM</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tbGrndCtrlIM">
         <attribute name="title">
          <string>Ground Control IM</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
